src/main/resources/hudson/os/windows/ManagedWindowsServiceLauncher/help_pt_BR.properties

```diff
@@ -22,6 +22,6 @@
 
 # Starts a Windows agent by <a href="http://en.wikipedia.org/wiki/Windows_Management_Instrumentation">a remote management facility</a> built into Windows. \
 # Suitable for managing Windows agents. \
-# Slaves need to be IP reachable from the master.
+# Agents need to be IP reachable from the controller.
 blurb=Iniciar um Windows agent <a href="http://en.wikipedia.org/wiki/Windows_Management_Instrumentation">uma ferramenta de gerenciamento remota</a> \
    agent precisam ter um IP alcan\u00e7\u00e1vel pelo master
```

Comment:
I don't speak Portuguese, but looks like you missed one.

Reply:
You're right. Fixed